lib/modules/showImages/expando.js

```diff
@@ -54,13 +52,13 @@ export class Expando {
 	static getEntryExpandoFrom(thing: ?Thing): ?Expando {
 		if (!thing) return null;
 
-		const button = $(thing.entry).find('> .expando-button')[0];
+		const button = thing.entry.querySelector('.expando-button');
 		if (!button) return null;
 
 		let expando = expandos.get(button);
 
 		if (!expando) {
-			const box = $(thing.entry).find('> .expando')[0];
+			const box = thing.entry.querySelector('.expando');
```

Comment:
This is safe because the entry doesn't include child comments, etc., so there can only be one `.expando`